code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -119,11 +119,11 @@
 	channels = list("Service" = 1)
 
 /obj/item/encryptionkey/headset_cent
-	name = "\improper CentCom radio encryption key"
-	desc = "An encryption key for a radio headset.  To access the CentCom channel, use :y."
+	name = "\improper Head Office radio encryption key"
+	desc = "An encryption key for a radio headset.  To access the Head Office channel, use :y."
 	icon_state = "cent_cypherkey"
 	independent = TRUE
-	channels = list("CentCom" = 1)
+	channels = list("Head Office" = 1)
```

Comment:
This should be using defines and that makes me sad. Thats out the scope of this PR though